ui/src/warp_runner/manager/commands/multipass_commands.rs

```diff
@@ -60,6 +60,21 @@ pub enum MultiPassCmd {
     },
 }
 
+// hide sensitive information from debug logs
+impl std::fmt::Display for MultiPassCmd {
```

Comment:
You should manually implement `Debug` too since that would also show the password.

Reply:
Just as a note, this is implemented on line 14.

Reply:
this turned out to be a lot easier than i thought it would be . `derive_more` is such an awesome crate.